packages/modular-scripts/src/utils/verifyPackageTree.ts

```diff
@@ -144,7 +144,7 @@ export async function verifyPackageTree(): Promise<void> {
               `P.S. We know this message is long but please read the steps above :-)\n`,
             ),
         );
-        process.exit(1);
+        throw new Error(`verify package tree failed`);
```

Comment:
`Verifying your package tree failed`?

Reply:
This is really an internal error to be picked up here: https://github.com/jpmorganchase/modular/pull/587/files#diff-bcc1fba74697e5316519a01ed00d33a3e255a7f74523c6076c3d08d97e39d176R122